frontend/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "freva2",
+  "version": "0.0.1",
+  "description": "Frontend for freva 2",
+  "main": "index.tsx",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack --mode=production --node-env=production",
+    "build:dev": "webpack --mode=development",
+    "dev": "webpack serve",
+    "lint": "eslint ."
+  },
+  "author": "",
+  "dependencies": {
+    "react": "18.2.0",
+    "react-dom": "18.2.0"
+  },
+  "devDependencies": {
+    "@loadable/component": "^5.15.2",
+    "@loadable/webpack-plugin": "^5.15.2",
+    "@types/node": "^18.11.9",
+    "@types/react": "^18.0.25",
+    "@types/react-dom": "18.0.9",
+    "@types/webpack": "^5.28.0",
+    "@typescript-eslint/eslint-plugin": "5.43.0",
+    "@typescript-eslint/parser": "5.43.0",
+    "autoprefixer": "10.4.13",
+    "css-loader": "^6.7.2",
+    "eslint": "^8.27.0",
+    "eslint-config-prettier": "^8.5.0",
+    "eslint-import-resolver-typescript": "3.5.2",
+    "eslint-plugin-import": "^2.26",
+    "eslint-plugin-react": "^7.31",
+    "eslint-webpack-plugin": "^3.2",
+    "mini-css-extract-plugin": "2.6.1",
+    "postcss": "^8.4.16",
+    "postcss-loader": "^7.0.1",
+    "prettier": "^2.7.1",
+    "sass": "^1.56.1",
+    "sass-loader": "^13.2.0",
+    "style-loader": "^3.3.1",
+    "ts-loader": "^9.4.1",
+    "typescript": "4.8.4",
+    "webpack": "^5.75.0",
+    "webpack-bundle-tracker": "^1.7.0",
+    "webpack-cli": "^4.10.0",
+    "webpack-dev-server": "^4.11.1",
+    "webpack-merge": "^5.8.0"
```

Comment:
I know we've had this discussion in the past. And my question also comes partially from the fact that I am unfamiliar with the npm package manager (or whatever this is based on). How do we make sure that the dependencies get updated? Please correct me if I am mistaken, but it would mean that fixing those dependencies could be troublesome in the long run. For example, if the dependencies are updated in like two years, we will have to rewrite many parts of the app because things will have changed.

Wouldn't it be better to relax those constraints and let a CI test suite (you're planning on adding unit tests for the front end, right) take care of it? If something breaks then, I would argue, fixing is still much easier than including a lot of changes caused by updates further down the road.

Reply:
If you want we can discuss this again on a wednesday in a bigger group. 

I _guess_ it will get a bit easier to keep track of changes with TypeScript than it is with plain JS but I am still very reluctant to loosen all constraints here. Even with a hypothetical 100% code coverage,  I don't want to fix a broken pipeline every second week and learning about the newest changes of library xy as my main job.

Reply:
Ok, how about fixing major or minor versions at least?

Reply:
btw dependabot also supports npm: https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file